### ISSUE TITLE
Change public entry

### ISSUE THREAD

**Issue by hl** [uberous/lista#12]
#### Should we remove `content` from the public entry?

The idea is that people can send in their entry and once the admin receives it he/she can write the content. Either by asking the sender some questions (interview) or writing something themselves.

#### Some kind of contact info

Should we add email or something to the public entry form so the admin can contact the sender if more info is required?

**Comment by jeroenvisser101** [comment]
Good call—I think we can leave the `content` field and add in a `name` and `email` field. We might even ask them about their insta-page (so the URL, and maybe a short description?)

**Comment by hl** [comment]
The instagram page could be pasted in the `link` field. I really would like to make this app as generic as possible while still good enough to be able to launch instahustlers.com

**Comment by jeroenvisser101** [comment]
Ah, the link field would do yeah, but some kind of 'intro' would make sense so admins can review faster?

**Comment by panoply** [comment]
Regarding this, is it not more viable to simply use a wrapper to connect to the users Instagram API and get this information? Here is a wrapper: [Elixtagram](https://github.com/Zensavona/elixtagram) 

@jeroenvisser101 agree, 'intro' would work.
